backstop/scenarios/radio-tile.js

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+const { scenario } = require('../scenarioHelper');
+
+module.exports = [
+  scenario('default', {
+    selectors: ['#demo-default'],
```

Comment:
Would be good to check hover, focus too

Reply:
Added tests for hover and focus. Not sure what this scenario should be called. default? Type default? All?

Reply:
COuld you take screenshot of whole group instead of single tile? So leave the selector in `actions` just change selector of whole group in `selectors`. Would be easier to see if focus and especially hover visuals are fine.

Reply:
Fixed